src/test/java/core/basesyntax/service/FruitShopServiceImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitRecord;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.operation.BalanceHandlerImpl;
+import core.basesyntax.service.operation.Handler;
+import core.basesyntax.service.operation.PurchaseHandlerImpl;
+import core.basesyntax.service.operation.ReturnHandlerImpl;
+import core.basesyntax.service.operation.SupplyHandlerImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitShopServiceImplTest {
+    private static List<FruitRecord> fruitRecordList;
+    private static FruitShopService fruitShopService;
+    private int expected;
+    private int actual;
+
+    @BeforeClass
+    public static void beforeClass() {
+        Map<Operation, Handler> handlerMap = new HashMap<>();
+        handlerMap.put(Operation.BALANCE, new BalanceHandlerImpl());
+        handlerMap.put(Operation.SUPPLY, new SupplyHandlerImpl());
+        handlerMap.put(Operation.PURCHASE, new PurchaseHandlerImpl());
+        handlerMap.put(Operation.RETURN, new ReturnHandlerImpl());
+        OperationStrategy operationStrategy = new OperationStrategyImpl(handlerMap);
+        fruitShopService = new FruitShopServiceImpl(operationStrategy);
+        Integer expected = Storage.storage.get(new Fruit("banana"));
```

Comment:
why do we need this here `Integer expected = Storage.storage.get(new Fruit("banana"));`?